app/src/main/contrib/components/Responsibilities/BurdenEstimates/UploadBurdenEstimatesForm.tsx

```diff
@@ -340,6 +332,5 @@ export const mapDispatchToProps:
 
 export const UploadEstimatesForm =
     compose<UploadEstimatesProps, UploadEstimatesPublicProps>(
-        connect(mapStateToProps, mapDispatchToProps),
-        branch((props: UploadEstimatesProps) => props.populatingInProgress, renderComponent(LoadingElement))
```

Comment:
This was causing the component to unmount/remount when `populatingInProgress` changed, hence why the fields were getting cleared. Have moved the loading spinner to just replace the buttons while populating is in progress, rather than replacing the whole component.)